README.md

```diff
@@ -2,38 +2,33 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-auditor-iam/workflows/build/badge.svg)](https://github.com/cisagov/cool-auditor-iam/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This project is used to manage IAM permissions for auditor users.
 
-## Usage ##
+## Pre-requisites ##
 
-```hcl
-module "example" {
-  source = "github.com/cisagov/cool-auditor-iam"
+The user accounts for all auditors must have been previously created (we
+recommend using the
+[`cool-users-non-admin`](https://github.com/cisagov/cool-users-non-admin)
+repository to create users).
 
-  aws_region            = "us-west-1"
-  aws_availability_zone = "b"
-  subnet_id             = "subnet-0123456789abcdef0"
+## Usage ##
 
-  tags = {
-    Key1 = "Value1"
-    Key2 = "Value2"
-  }
-}
-```
+1. Create a Terraform workspace (if you haven't already done so) by running
+   `terraform workspace new <workspace_name>`
+1. Create a `<workspace_name>.tfvars` file with all of the required
+  variables (see [Inputs](#Inputs) below for details):
 
-## Examples ##
+  ```hcl
+  auditors = {
+    "first1.last1"    = { "roles" = ["security_audit"] },
```

Comment:
```suggestion
    "firstname1.lastname1"    = { "roles" = ["security_audit"] },
    "firstname2.lastname2"    = { "roles" = ["financial_audit"] },
    "firstname3.lastname3"    = { "roles" = ["financial_audit", "security_audit"] },
```

I suggest we mirror the labeling in https://github.com/cisagov/cool-users-non-admin/pull/13